GirafRest/Controllers/PictogramController.cs

```diff
@@ -276,21 +276,44 @@ public async Task<Response<WeekPictogramDTO>> SetPictogramImage(long id)
             //Update the image
             byte[] image = await _giraf.ReadRequestImage(HttpContext.Request.Body);
 
-            string path = imagePath + pictogram.Id + ".png";
-        
+            string originalPath = imagePath + pictogram.Id + ".png";

```

Comment:
maybe put a comment on why you use temp and old variables, so that they won't have to figure it out themselves in the future